components/navbar/NavbarExploreOptions.vue

```diff
@@ -15,30 +15,31 @@
       </span>
     </span>
     <hr aria-role="menuitem" class="dropdown-divider my-4" />
-    <span
-      v-for="option in availableChains.slice(0, 3)"
-      :key="option.value"
-      :class="[
-        'menu-item',
-        'mr-2',
-        { 'is-active': selectedChain === option.value },
```

Comment:
No more active state?

Reply:
yeah @exezbcz asked to remove active from explore dropdown

Reply:
Aight understandable